msaad.go

```diff
@@ -913,6 +946,7 @@ func (m *MSAAD) getBearerToken() (token string, expiresAt time.Time, err error)
 	}
 
 	token = tokenJSON.AccessToken
+	// can time.Duration(string) not PANIC?
```

Comment:
`tokenJSON.ExpiresIn` is an `int`. Go lang would not allow you to put anything in there that is not an `int`. Golang also sets default values.